HPXMLtoOpenStudio/resources/defaults.rb

```diff
@@ -5736,6 +6136,628 @@ def self.get_ceiling_fan_months(weather)
     return months
   end
 
+  # Returns the number of breaker spaces based on rated power and voltage.
+  #
+  # @param watts [Double] power rating (W)
+  # @param voltage [String] '120' or '240'
+  # @param max_amps [Double] maximum amperage (A)
+  # @return [Integer] the number of breaker spaces
+  def self.get_breaker_spaces_from_power_watts_voltage_amps(watts, voltage, max_amps)
+    return 0 if watts == 0
+
+    # Note that default_panels.csv has a Breaker Spaces column manually populated based on the following calculation.
+    # If max_amps were to change, for example, the value in Breaker Spaces may change.
+    required_amperage = watts / Float(voltage)
+    num_branches = (required_amperage / max_amps).ceil
+    num_breakers = num_branches * Integer(Float(voltage) / 120)
+    return num_breakers
+  end
+
+  # Gets the default voltage for a branch circuit based on attached component.
+  #
+  # @param branch_circuit [HPXML::BranchCircuit] Object that defines a single electric panel branch circuit
+  # @return [String] '120' or '240'
+  def self.get_branch_circuit_voltage_default_values(branch_circuit)
+    branch_circuit.components.each do |component|
+      if component.is_a?(HPXML::HeatingSystem)
+        if component.heating_system_fuel == HPXML::FuelTypeElectricity
+          return HPXML::ElectricPanelVoltage240
+        end
+      elsif component.is_a?(HPXML::CoolingSystem)
+        if component.cooling_system_type != HPXML::HVACTypeRoomAirConditioner
+          return HPXML::ElectricPanelVoltage240
+        end
+      elsif component.is_a?(HPXML::HeatPump)
+        if component.heat_pump_fuel == HPXML::FuelTypeElectricity
+          return HPXML::ElectricPanelVoltage240
+        end
+      elsif component.is_a?(HPXML::HVACDistribution) ||
+            component.is_a?(HPXML::ClothesWasher)
+        component.is_a?(HPXML::Oven) ||
```

Comment:
Missing a || here. Also, some of these don't make any sense to me. Why would a clothes washer, refrigerator, or freezer be 240V?